apps/s_client.c

```diff
@@ -2389,9 +2397,12 @@ int s_client_main(int argc, char **argv)
         }
         break;
     case PROTO_CONNECT:
-        if (!OSSL_HTTP_proxy_connect(sbio, host, port, proxyuser, proxypass,
-                                     0 /* no timeout */, bio_err, prog))
-            goto shut;
+	{
```

Comment:
No need to add a block - there are no declarations here.

Reply:
I did put in some declarations then removed them, block now removed.